### DISCUSSION TITLE
posts/come-on-paper/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#40] [Comments]
# posts/come-on-paper/

受《王川宝典》提醒，俺准备读起英文论文来啦，和小王一起。
[&hellip;] 目前拥有良好的论文阅读条件：
[&hellip;] 起步阶段每人每周自选一篇，不限主题，读后交流输出。俺计划从 Data Science 开始，广度优先，后面或许会到心理学、营养学等。
[&hellip;] 输出方面，原本计划放在学习博客上，考虑到更新频率和渲染速度，最终决定在本站新增栏目，一直更新下去。 &hellip;

https://shitao5.org/posts/come-on-paper/

**Comment by hongtaoh** [comment]
每次来你这里都要吃一顿狗粮

**Reply by Shitao5** [reply]
就当鸿涛沉浸于自然科学时的小憩嘛😊
